src/main/python/main/ayab/preferences.py

```
@@ -129,6 +130,12 @@ class Preferences(SignalSender):
         "lower_display_stitch_width": int,
     }
 
+    """How many recent files can be added to menu."""
+    MAX_RECENT_COUNT = 5
+
+    """Recent files list"""
+    recentFiles = []
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)